htdocs/modules/system/admin/modulesadmin/modulesadmin.php

```
@@ -119,11 +116,16 @@ function xoops_module_install($dirname)
                 } else {
                     $msgs[] = '<p>' . sprintf(_AM_SYSTEM_MODULES_SQL_FOUND, "<strong>{$sql_file_path}</strong>") . '<br  />' . _AM_SYSTEM_MODULES_CREATE_TABLES;
                     include_once XOOPS_ROOT_PATH . '/class/database/sqlutility.php';
-                    $sql_query = fread(fopen($sql_file_path, 'r'), filesize($sql_file_path));
+                    $sql_query = file_get_contents($sql_file_path);
```

PHPMD (2.15.0)

[error] The variable $sql_query is not named in camelCase. (undefined)

(CamelCaseVariableName)